packages/web-app-files/tests/unit/views/spaces/__snapshots__/Projects.spec.ts.snap

```diff
@@ -8,188 +8,44 @@ exports[`Projects view different files view states lists all available project s
-                                    <!---->
-                                    <!---->
-                                  </button></li>
-                              </ul>
-                              <ul id="oc-files-context-actions-secondaryActions" class="oc-list oc-my-rm oc-mx-rm oc-files-context-actions oc-pb-s oc-pt-s oc-files-context-actions-border">
-                                <li class="oc-files-context-action oc-px-s oc-rounded oc-menu-item-hover"><button type="button" class="oc-button oc-rounded oc-button-s oc-button-justify-content-center oc-button-gap-m oc-button-passive oc-button-passive-raw oc-files-actions-disable-trigger action-menu-item" data-testid="action-handler"><span class="oc-icon oc-icon-m oc-icon-passive" data-testid="action-icon"><!----></span> <span data-testid="action-label" class="oc-files-context-action-label">Disable</span>
-                                    <!---->
-                                    <!---->
-                                  </button></li>
-                                <li class="oc-files-context-action oc-px-s oc-rounded oc-menu-item-hover"><button type="button" class="oc-button oc-rounded oc-button-s oc-button-justify-content-center oc-button-gap-m oc-button-passive oc-button-passive-raw oc-files-actions-restore-trigger action-menu-item" data-testid="action-handler"><span class="oc-icon oc-icon-m oc-icon-passive" data-testid="action-icon"><!----></span> <span data-testid="action-label" class="oc-files-context-action-label">Enable</span>
-                                    <!---->
-                                    <!---->
-                                  </button></li>
-                              </ul>
-                              <ul id="oc-files-context-actions-trashBin" class="oc-list oc-my-rm oc-mx-rm oc-files-context-actions oc-pt-s">
-                                <li class="oc-files-context-action oc-px-s oc-rounded oc-menu-item-hover"><button type="button" class="oc-button oc-rounded oc-button-s oc-button-justify-content-center oc-button-gap-m oc-button-passive oc-button-passive-raw oc-files-actions-delete-trigger action-menu-item" data-testid="action-handler"><span class="oc-icon oc-icon-m oc-icon-passive" data-testid="action-icon"><!----></span> <span data-testid="action-label" class="oc-files-context-action-label">Deleted files</span>
-                                    <!---->
-                                    <!---->
-                                  </button></li>
-                              </ul>
-                            </div>
-                            <!---->
-                            <!----> <input id="space-image-upload-input" type="file" name="file" multiple="multiple" tabindex="-1" accept="image/*">
-                          </div>
+                          <space-context-actions-stub space="undefined" items="undefined"></space-context-actions-stub>
```

Comment:
Are these changes in this file expected?
`undefined` here doesn't look good tbh

Reply:
As discussed: Seems to be a weird behaviour of the vue jest utils. I added a comment for now.